.github/workflows/CD.yml

```diff
@@ -340,7 +352,7 @@ jobs:
         id: version
         run: |
           TAG=`cat build/packages/VERSION`
-          TARGET_VERSION=$TAG bash adot/tools/workflow/docker-version-compare.sh
+          TARGET_VERSION=$TAG bash main/tools/workflow/docker-version-compare.sh
```

Comment:
If you're going to change this to use the main branch instead of the input SHA, you should also change the path for the other tools being used.

Reply:
The only used tool with the path ```adot``` is the releasing of dot operator. Other steps are using cache with build/packages. So I think it should be fine.

Reply:
That is not true. On line 370, the image mirror is using the `adot` path.

Reply:
Yeah you are right. I misunderstood the use of the image mirror. Fixed it.

Reply:
Also if that's the case, then we don't need to checkout for the input SHA anymore. Nice catch.